adapter.go

```diff
@@ -184,8 +184,8 @@ func openDBConnection(driverName, dataSourceName string) (*gorm.DB, error) {
 		db, err = gorm.Open(postgres.Open(dataSourceName), &gorm.Config{})
 	} else if driverName == "mysql" {
 		db, err = gorm.Open(mysql.Open(dataSourceName), &gorm.Config{})
-	//} else if driverName == "sqlite3" {
-	//	db, err = gorm.Open(sqlite.Open(dataSourceName), &gorm.Config{})
+		//} else if driverName == "sqlite3" {
```

Comment:
Why change it.

Reply:
The change of these lines's is just for code format.